jumanji/training/configs/env/tetris.yaml

```
@@ -0,0 +1,26 @@
+name: tetris
+registered_version: Tetris-v0
+
+network:
+    conv_num_channels: 32
+    tetromino_layers: 16
+    final_layer_dims: [64, 40]
+
+training:
+    num_epochs: 1000
+    num_learner_steps_per_epoch: 10
+    n_steps: 20
+    total_batch_size: 64
+
+evaluation:
+    eval_total_batch_size: 100
+    greedy_eval_total_batch_size: 100
```

We probably want something a bit bigger to reduce the variance of the estimate.
```suggestion
    eval_total_batch_size: 1000
    greedy_eval_total_batch_size: 1000
```